### PR TITLE
[ROAD-433] Fix: overview dark theme support

### PR DESCRIPTION
**Pull request by aldanchenko**
Fixed issue with white text on white background. 
Additional: fixed black text for dark theme.

### CODE REVIEW

File: Snyk.VisualStudio.Extension/Snyk/VisualStudio/Extension/UI/Toolwindow/OssDescriptionControl.xaml.cs [hunk 1]
```diff
@@ -51,6 +51,14 @@ public Vulnerability Vulnerability
             }
         }
 
+        /// <summary>
+        /// Adapt components for VS theme change.
+        /// </summary>
+        public void AdaptComponentsForThemeChange()
+        {
+            this.overview.AdaptForeground();
```
Comment (bastiandoetsch) [inline Snyk.VisualStudio.Extension/Snyk/VisualStudio/Extension/UI/Toolwindow/OssDescriptionControl.xaml.cs:59]: What does AdaptForeground() do?

File: Snyk.VisualStudio.Extension/Snyk/VisualStudio/Extension/UI/Tree/TreeNode.cs [hunk 2]
```diff
@@ -29,6 +29,11 @@ public class TreeNode
         /// </summary>
         public virtual bool IsRoot => false;
 
+        /// <summary>
+        /// Gets a value indicating whether is node enabled/disabled.
+        /// </summary>
+        public virtual bool Enabled => true;
+
```
Comment (bastiandoetsch) [inline Snyk.VisualStudio.Extension/Snyk/VisualStudio/Extension/UI/Tree/TreeNode.cs:36]: How does this relate to the PR?

Comment (aldanchenko) [reply Snyk.VisualStudio.Extension/Snyk/VisualStudio/Extension/UI/Tree/TreeNode.cs:36]: Ah... sorry, this doesn’t apply directly to this PR

**Comment by aldanchenko** [comment]
Thanks @bastiandoetsch!

**Pull request merged**